### ISSUE TITLE
Add tips for accessible document creation

### ISSUE THREAD

**Issue by ericmayes** [UCSantaBarbara/webguide#57]
One big thing missing from our Accessibility recommendations are tips for creating accessible documents (PDF, Word docs, etc). I attended a webinar where I saw a few slides that discuss the process from different applications. These still have to be tested out, but I'm pretty sure they work. Not sure if the group would want to decide to include this type of recommendation, but if so, we could make accessible HTML versions of the text on these slides to help the campus create accessible documents.

![demystifying-wcag-20-an-intro-to-web-office-indesign-pdf-accessib](https://cloud.githubusercontent.com/assets/7233610/8558504/822ebdea-24b9-11e5-850e-6fe60247641e.jpg)
![demystifying-wcag-20-an-intro-to-web-office-indesign-pdf-accessibility-55-638](https://cloud.githubusercontent.com/assets/7233610/8558506/823225e8-24b9-11e5-996c-c18701407ff2.jpg)
![demystifying-wcag-20-an-intro-to-web-office-indesign-pdf-accessibility-56-638](https://cloud.githubusercontent.com/assets/7233610/8558505/82318f70-24b9-11e5-860c-19e475da53c2.jpg)
![demystifying-wcag-20-an-intro-to-web-office-indesign-pdf-accessibility-57-638](https://cloud.githubusercontent.com/assets/7233610/8558508/82354dfe-24b9-11e5-9226-99952d3e5ceb.jpg)
![demystifying-wcag-20-an-intro-to-web-office-indesign-pdf-accessibility-58-638](https://cloud.githubusercontent.com/assets/7233610/8558507/823337f8-24b9-11e5-80bb-dfce44a85163.jpg)

**Comment by UCEAP-JCOLON** [comment]
Excellent.
Thanks Eric!

JAMES cOLón
WebMaster
University of California Education Abroad Program
805 893 7268 | eap.ucop.eduhttp://uc.eap.ucop.edu/
UCEAP Systemwide Office
6950 Hollister Avenue, Suite 200
Goleta CA 93117-5823
[http://vmoss-intra.uceap.local/Documents/brochures/Facebook_for_Office.png]https://www.facebook.com/UCEAP

[http://vmoss-intra.uceap.local/Documents/brochures/Twitter_for_Office.png]https://twitter.com/UCEAP

[http://vmoss-intra.uceap.local/Documents/brochures/Pinterest_for_Office.png]https://www.pinterest.com/uceapabroad/

[http://vmoss-intra.uceap.local/Documents/brochures/Youtube_for_Office.png]https://www.youtube.com/user/UCEAP

[http://vmoss-intra.uceap.local/Documents/brochures/Instagram_for_Office.png]https://instagram.com/uceap/

[http://vmoss-intra.uceap.local/Documents/brochures/Blog_for_Office.png]http://blog.eap.ucop.edu/

**Comment by tenken** [comment]
I have created a Wiki page for [Accessible Dcoument Tips](../wiki/Accessible-Document-Tips) in which I attempted to convert all this to HTML. Feel free to edit and correct the page as needed.

I added a section on like proper use of Word Processors.

An interesting side note is this only covers like PDF generation, not say providing a PowerPoint file to end-users, or a normal Word file which is what I created a section for. Also while an accessible PDF may want a 1st page of the document title -- few real world PDFs i've _ever_ linked on a website for an event such as a Flyer or other promotional material _ever_ do this. This seems like a recommendation for a course material pdf, lab workbook, etc.

The purpose of the 1st page would be (I suppose) to announce the title of the PDF so it just doesnt start on content -- but 1 could argue a proper _filename_ on the PDF is just as expressive and useful (god I hate grad003-2015(4).pdf filenames).

Feedback appreciated.

**Comment by loganfranken** [comment]
Thanks for bringing this up @ericmayes! And thanks for moving this into the wiki, @tenken!

@tenken, I made a few (minor) revisions:
https://github.com/ucsb-wsg/ucsb-wsg.github.io/wiki/Accessible-Document-Tips/_compare/9d9a10f5b1e1d8047b9d1a46883778167d358ba2...660413c70072b0c0442dfcbb6efc272fc2a03b76

I think we're good to close this issue since @tenken did such a good job of moving over the content from the slides.

Did you want to create a separate issue for PDF accessibility issues, @tenken?

**Comment by loganfranken** [comment]
Are we good to close this, @tenken @ericmayes @UCEAP-JCOLON?
